### PR TITLE
Added week 1 post

### PR DESCRIPTION
**Pull request by m-agour**
Update 2: It's done.
Update: This PR still needs some improvements and review from my end and will do this by today.

My blog post for the first week.

### CODE REVIEW

File: docs/source/posts/2022/2021-06-08-week-1-mohamed.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+--------------------------
+
+I continued implementing the Timeline class. Added some functionalities according to what we discussed in the weekly meeting, such as handling Fury Actors, adding, removing, translating, scaling them according to the keyframes. Then made two tutorials for it with simple UI to control the playback of the animation.
+
+  .. raw:: html
+
+        <iframe id=""player" type="text/html" width="640" height="390" src="https://user-images.githubusercontent.com/63170874/174503916-7ce0554b-9943-43e3-9d5c-c97c9ce48eaf.mp4" frameborder="0"></iframe>
+
+
+Reviewed Bruno's PR `#424`_ as Filip advised to help me figure out how to change uniforms' values during runtime. I found it straightforward. I also found a similar method already used in some tutorials and experiments to set the time uniform, which is based on the same idea. They both are using shaders callback functions.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2021-06-08-week-1-mohamed.rst:28]: typo semicolon `'` after uniforms

File: docs/source/posts/2022/2021-06-08-week-1-mohamed.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Going through the VTk's documentations, I found a more efficient way to do this. It's newly implemented in VTK 9, does not require callbacks, and is easy to use on individual actors. I had to verify first that Fury does not support an older version of VTK and it does not. I tested it and ran into some issues so As Filip instructed I don't waste a lot of time with uniforms so I postponed it for the next week.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2021-06-08-week-1-mohamed.rst:29]: Capital `K` in the `VTk's`

**Review by ganimtron-10 — COMMENTED** [review]
Nice Blog @m-agour .
Also the blog was successfully generated.

Please take a look at the below comments.
Thanks!

**Review by xtanion — COMMENTED** [review]
Hi @m-agour , Nice blog and blogs are also generating fine 👍🏽 . Please take a look at my comment below.
Thanks.

**Comment by skoudoro** [comment]
What is the status of this PR?

**Comment by skoudoro** [comment]
LGTM, merging

**Pull request merged**